src/main/java/org/jenkinsci/maven/plugins/hpi/AbstractHpiMojo.java

```diff
@@ -118,6 +118,12 @@
     @Parameter
     private String compatibleSinceVersion;
 
+    /**
+     * Optional - specify the minimum version of Java that this plugin requires.
+     */
+    @Parameter
+    private String javaVersion;
```

Comment:
should this be called `mininumJavaVersion`, as with JDK9, 10 and upcoming, we might have to introduce some `maxJavaVersion` or so? (Hopefully not, but let's still try and embed that "minimum" version implied meaning into the prop name?)